cmd/twosum/main_test.go

```diff
@@ -55,14 +67,54 @@ func Test_doubleSumMap(t *testing.T) {
 	}
 }
 
+func getTestData() *TestData {
+	once := &sync.Once{}
+	once.Do(func() {
+		m := make(map[int]bool)
+		sums := make(map[int]bool)
+		nums := make([]int, 2000)
+		var target int
+		r := rand.New(rand.NewSource(time.Now().Unix()))
+		for len(m) < 2000 {
+			m[r.Intn(3000)] = true
+		}
+		i := 0
+		for key, _ := range m {
+			nums[i] = key
```

Comment:
You can use `rand.Perm` to generate a slice of random integers.

Reply:
thanks for that, really no need unique integers in test slice